build.gradle

```diff
@@ -10,6 +10,8 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath "com.google.dagger:hilt-android-gradle-plugin:$hilt_version"
         classpath 'com.google.gms:google-services:4.3.10'
+        def nav_version = "2.4.1"
```

Comment:
You should write the version where kotlin and hilt version are : ext.nav_version = '2.4.1'

Reply:
Actually you can define it here, I don't think it changes a lot. What's the difference?

Reply:
I already defined two versions in the other way :) Let's keep everything coherent.